### PR TITLE
✨ Extension for flowplayer

### PR DESCRIPTION
**Pull request by okkonen**
PR implements an extension for [Flowplayer](http://www.flowplayer.com) video player

### CODE REVIEW

**Comment by alanorozco** [comment]
Thank you for your contribution.

Only constructing an iframe is not enough to warrant a player component. Please use `amp-video-iframe` or `amp-iframe`.

**Comment by okkonen** [comment]
> Thank you for your contribution.
> 
> Only constructing an iframe is not enough to warrant a player component. Please use `amp-video-iframe` or `amp-iframe`.

Can you elaborate this a bit more? I have used `amp-jwplayer` as a reference. It's very similar service as what Flowplayer provides. I also recall that it was mentioned somewhere in the documentation as a reference extension for this kind of services, just cannot find it any more. There are also number of similar extensions implemented in same way: amp-kalture-player, amp-springboard-player, amp-reach-player, amp-hulu etc.

Other option that many player are using seems to be like this:
```
  const frame = htmlFor(
    element
  )`<iframe frameborder=0 allowfullscreen></iframe>`;
````

**Comment by okkonen** [comment]
@alanorozco please check my previous comment.

**Comment by nainar** [comment]
@alanorozco

File: extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```
Comment (Gregable) [inline extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html]: Please add the copyright statement to the top of this test file. You can look at any other validator test for a similar statement.

Comment (okkonen) [reply extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html]: Copyright statement added

Comment (Gregable) [reply extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html]: You will need to rerun the validator update_tests command to pick the same up in the corresponding `.out` file.

Comment (okkonen) [reply extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html]: I also run the validator to update .out file.

Comment (okkonen) [reply extensions/amp-flowplayer/0.1/test/validator-amp-flowplayer.html]: @Gregable please re-review

**Comment by JDevlin88** [comment]
@alanorozco or @Gregable Could you please check out the above :)

**Comment by alanorozco** [comment]
@JDevlin88 @okkonen 

Historically added components that fulfill one particular use case (e.g. `amp-jwplayer`) that can now be fulfilled with something generic (e.g. `amp-iframe` or `amp-video-iframe`) do not justify future componetns.

All this seems to do is create a frame with a simple URL scheme based on a few params. This does not warrant a specific component.

New components have a fixed maintenance and bloat cost. Please explain why authors can't simply use an `amp-iframe` or an `amp-video-iframe` instead.

**Comment by alanorozco** [comment]
(With regards to other 3rd party players, they actually talk to specific Javascript APIs inside the frame and they interact with the AMP-level `VideoManager` for video features. This component does neither).

**Comment by mrjoro** [comment]
Closing this PR since it no longer seems active.  Please reopen if needed.

**Pull request closed without merging**